config/zones/SK.yaml

```diff
@@ -55,13 +55,19 @@ emissionFactors:
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 1300.76
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 764.9
```

Comment:
Nováky power plant had an emission factor of 2000+, which seems too unrealistic so it has been discarded.
The two coal power plants in Slovakia has both been closed now (in Dec 2023 and April 2024).